concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -19,4 +35,36 @@ def handler(event, context):
         },
     )
 
+    song_list_update_response = update_song_list(message)
```

Comment:
Neither this variable nor `sns_response` are being used.

Reply:
Yep, just seemed like we were following a pattern within the handlers of `service_response = serviceCall()`. Removed.